content/en/docs/setup/independent/setup-ha-etcd-with-kubeadm.md

```diff
@@ -107,7 +107,7 @@ this example.
 
     If you already have a CA then the only action that is copying the CA's `crt` and
     `key` file to `/etc/kubernetes/pki/etcd/ca.crt` and
-    `/etc/kubernetes/pki/etcd/ca.key`. After those files have been copied, please
+    `/etc/kubernetes/pki/etcd/ca.key`. After those files have been copied, 
     proceed to "Create certificates for each member".
```

Comment:
One more, link to this step instead of quoting the title.

Reply:
Sure, or just "continue to the next step"? If it is literally the next step, that seems adequate.